apps/admin/src/app/pages/user/user-change-password.tsx

```diff
@@ -0,0 +1,144 @@
+import { Icon } from '@admin/app/components';
+import { errorMessage, useAuth, useBoolean, UserService, useToasty } from '@libs/react-core';
+import { Box, Button, Card, CardContent, CardHeader, Container, IconButton, InputAdornment, Stack, useTheme } from '@mui/material'
+import { Field, Form, Formik, FormikHelpers } from 'formik'
+import { TextField } from 'formik-mui';
+import { useCallback } from 'react'
+import { object, ref, string } from 'yup';
+
+const userService = UserService.getInstance<UserService>();
+
+
+const validationSchema = object().shape({
+    oldPassword: string().label('Old Password').required(),
+    password: string().label('New Password').required().matches(
+        /^(?=.*[a-z])(?=.*[0-9])(?=.{8,})/,
+        "Must Contain 8 Characters, One Uppercase, One Lowercase, One Number and One Special Case Character"
+    ),
+    confirmPassword: string().label('Confirm Password').oneOf([ref('password'), null], 'Passwords must match').required(),
+});
+
+const defaultValues = {
+    oldPassword: '',
+    password: '',
+    confirmPassword: '',
+};
+
+const UserChangePassword = () => {
+    const { user } = useAuth();
+    const showPassword = useBoolean()
+    const showOldPassword = useBoolean()
+    const confirmShowPassword = useBoolean()
+    const { showToasty } = useToasty()
+    const theme = useTheme()
+
+    const handleSubmitForm = useCallback(
+        (values, action: FormikHelpers<any>) => {
+            userService.changePassword(values).then(() => {
+                action.setSubmitting(false)
+                action.resetForm()
+                showToasty('Password Successfully updated')
+            }).catch((error) => {
+                action.setErrors({ afterSubmit: errorMessage(error) });
+                action.setSubmitting(false)
+                showToasty(error.message, 'error')
+            });
+        }, []
+    )
+
+    return (
+        <Container maxWidth='sm'>
+            <Card>
+                <CardHeader title='Change Password' />
+                <CardContent>
+                    <Formik
```

Comment:
We will use hook-form so once my PR is marge need to use hook-form. So note down this change

https://github.com/ack-solutions/nest-react-next/pull/4